scripts/shell.nix

```diff
@@ -15,6 +15,7 @@ mkShell {
     rename
     gnupg
     dput
+    copr-cli
```

Comment:
why are you adding this? its not mentioned anywhere else

Reply:
Oh, look like I've added it accidentally

Reply:
Removed